src/integrations/Braze/browser.js

```diff
@@ -96,9 +97,8 @@ class Braze {
     })(window, document, "script");
 
     window.braze.initialize(this.appKey, {
-      enableLogging: true,
+      enableLogging: this.enableLoggingToConsole,
```

Comment:
```suggestion
      enableLogging: this.enableLoggingToConsole || true,
```

Reply:
by default, this value will be false as it's boolean

Reply:
This destination is already used and an old one. If you make it false using a config variable then it will not be reflected in all the customers. So this variable will not be available for old customers.

Reply:
I think it's better to not show braze logs to end user. will update as you suggested

Reply:
Make sense. Verify one thing, if you are sending undefined in place of `enableLogging`, is the SDK handling this or not?

Reply:
updated